internal/controller/vrg_volrep.go

```diff
@@ -1918,7 +1918,7 @@ func (v *VRGInstance) addProtectedAnnotationForPVC(pvc *corev1.PersistentVolumeC
 		// v.updateProtectedPVCCondition(pvc.Name, PVCError, msg)
 		log.Error(err, "Failed to update PersistentVolumeClaim annotation")
 
-		return fmt.Errorf("failed to update PersistentVolumeClaim (%s/%s) annotation (%s) belonging to"+
+		return fmt.Errorf("failed to update PersistentVolumeClaim (%s/%s) annotation (%s) belonging to "+
```

Comment:
Turns out this log has

> {"VolumeReplicationGroup": {"name":"appset-deploy-rbd-busybox","namespace":"e2e-appset-deploy-rbd-busybox"}, "rid": "93ab4d6f-3f56-402c-ba11-90c7793ee00e", "State": "primary"}

So we can remove the "belonging to VolumeReplicationGroup (%s/%s)" part instead of fixing it.

Reply:
But the log does not help since this error is logged from another logger that does not have the vrg info.